app/models/integrations/app.rb

```diff
@@ -64,6 +76,10 @@ def self.slack_integration_url
     "#{ENV.fetch('FRONTEND_URL', nil)}/app/accounts/#{Current.account.id}/settings/integrations/slack"
   end
 
+  def self.linear_integration_url
+    "#{ENV.fetch('FRONTEND_URL', nil)}/linear/callback"
```

Comment:
Slack has the account id in the callback url. Is it not possible for linear? That we can get rid of the `encode_state` part

Reply:
Linear does not allow passing parameters in the redirect URL, so the redirect URL and the URL configured in the Linear app should be the same.